met-api/src/met_api/models/comment.py

```diff
@@ -101,11 +101,14 @@ def __create_new_comment_entity(comment):
         )
 
     @classmethod
-    def add_all_comments(cls, comments: list) -> DefaultMethodResult:
+    def add_all_comments(cls, comments: list, session=None) -> DefaultMethodResult:
         """Save comments."""
         new_comments = [cls.__create_new_comment_entity(comment) for comment in comments]
-        db.session.add_all(new_comments)
-        db.session.commit()
+        if session is None:
+            db.session.add_all(new_comments)
+            db.session.commit()
+        else:
+            session.add_all(new_comments)
```

Comment:
```
    if session is None:
            db.session.add_all(new_comments)
            db.session.commit()
        else:
            session.add_all(new_comments)
  ```
  to 
  ```
      db.session.add_all(new_comments)
      if session is None:
            db.session.commit()
            ```          
Does this work? the intention is that , if there is a session , dont commit rite? so this simplification might work

Reply:
It works as today because the session object is the same as the db.session, if we decide to create a new scoped session in the future I think that could cause a problem

Reply:
Makes sense..